specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -9185,6 +9256,64 @@
         "modelAsString": true
       }
     },
+    "IntegrationServiceEnvironmentManagedApiListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApi"
+          },
+          "description": "The integration service environment managed APIs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "The list of integration service environment managed APIs."
+    },
+    "IntegrationServiceEnvironmentManagedApi": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApiProperties",
+          "description": "The integration service environment managed api properties."
+        }
+      },
+      "description": "The integration service environment managed api.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "IntegrationServiceEnvironmentManagedApiProperties": {
+      "type": "object",
+      "description": "The integration service environment managed api properties.",
+      "properties": {
+        "deploymentParameters": {
+          "$ref": "#/definitions/IntegrationServiceEnvironmentManagedApiDeploymentParameters",
+          "description": "The integration service environment managed api deployment parameters."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ApiResourceProperties"
+        }
+      ]
+    },
+    "IntegrationServiceEnvironmentManagedApiDeploymentParameters": {
+      "type": "object",
+      "description": "The integration service environment managed api deployment parameters.",
+      "properties": {
+        "contentLinkDefinition": {
+          "$ref": "#/definitions/ContentLink",
```

Comment:
May I ask whether all sub properties under `ContentLink` need to be set in request body? I assume some sub properties are read-only, right? If yes, could you add read-only attribute for them?

![image](https://user-images.githubusercontent.com/19754191/135012480-31ebd7e7-dc6c-49e4-ac9f-955bc2799f55.png)

Reply:
You can set all of them if needed. But Uri is the only required one. I can add Required for Uri. Would that be enough?

Reply:
Sure. Thanks.

Reply:
Let me add all the other properties as readonly apart from the Uri.